common-lib/src/main/scala/com/gu/mediaservice/lib/aws/ThrallMessageSender.scala

```diff
@@ -43,7 +43,13 @@ object UpdateMessage {
         (__ \ "usageNotice").readNullable[UsageNotice] ~
         (__ \ "edits").readNullable[Edits] ~
         // We seem to get messages from _somewhere which don't have last modified on them.
-        (__ \ "lastModified").readNullable[DateTime].map(_.getOrElse(DateTime.now(DateTimeZone.UTC))) ~
+        (__ \ "lastModified").readNullable[DateTime].map{ d => d match {
+          case Some(date) => date
+          case None => {
+            logger.warn("Message received without a last modified date", __.toJsonString)
```

Comment:
Would be ecstatic if we could pull subject to its own log marker but

Reply:
Well, we sort of could, but I thought dumping the whole thing was probably useful.